### DISCUSSION TITLE
Can you make the military not shoot at you if you have the insergint profeson  or the clothes I would like that

### DISCUSSION THREAD

**Discussion by yourlocalwallhammer** [TEHE-Studios/ExpandedHelicopterEvents#228] [Q&A]
is this alredy a thing?

**Comment by thatH3ath3n** [comment]
i wish if you had a class that has/had military association (veteran/insurgent/ect) they would not shoot you

**Comment by sharkster91** [comment]
We are not doing it. You are welcome to make a submod but we arent going to do it.

**Comment by Chuckleberry-Finn** [comment]
The military not shooting you if you have military/police clothing on is already a thing, yes. Having the profession of veteran means nothing, neither does having a custom profession from another mod. As Shark stated, that person is free to add support on their end if they wish to.
